### ISSUE TITLE
Some troubleshooting process that may be helpful for others

### ISSUE THREAD

**Issue by yangsheng6810** [vace117/calibre-dedrm-docker-image#2]
Many thanks for your work!

I had trouble using it, but managed to figure it out. The process was not smooth, so I think it may help others by sharing the process (it would be great if you can include something in a troubleshooting section in the README file). Here is the problem I face and how I fixed it.

I ran `./launch_book_manager.sh` as instructed. While README suggests some wizard windows popping up, I didn't observe one. This is the source of the troubles, but I didn't realize at this time. After a few confirmations on command line, I was left at a bash prompt. I tried open `calibre`, and it cashed with the following log:
```
No protocol specified
qt.qpa.xcb: could not connect to display unix:0
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, xcb.

Aborted (core dumped)
```
It turns out the application cannot connect to the display. I managed to fix the problem by running the following on the host, according to [ref](https://github.com/jessfraz/dockerfiles/issues/329#issuecomment-368262183).
```shell
xhost +"local:docker@"
```
Then I am able to use `calibre`, but I cannot run `kindle`. It turns out Kindle for PC was never installed. I guess it was because the installer could not connect to display, thus not run. I installed it by running the actual installer with wine, and everything is fine now.

In conclusion, I propose adding a bullet point in the **Installation** section, suggesting running `xhost +"local:docker@"`  on the host before installation, so that all these troubles can be avoided.

**Comment by vace117** [comment]
Thank you, yangsheng6810! 

I've added your solution to the `README`:
https://github.com/vace117/calibre-dedrm-docker-image/blob/master/README.md#installation
